### PR TITLE
Fix broken platform elements

### PR DESCRIPTION
**Pull request by lockieRichter**
Updates the Dante and Platform components to use the `flutter_platform_widgets` package for convenience (https://pub.dev/packages/flutter_platform_widgets).

Also fixes a bunch of minor linting issues Dart was complaining about.

### CODE REVIEW

File: lib/src/ui/core/dante_components.dart [hunk 1]
```diff
@@ -1,111 +1,97 @@
 import 'package:flutter/material.dart';
 
-class DanteComponents {
-  DanteComponents._();
+Divider danteDivider(
```
Comment (lockieRichter) [inline lib/src/ui/core/dante_components.dart]: Not sure if we should keep these as functions or convert them to actual widgets. What do you think?

File: pubspec.yaml [hunk 2]
```diff
@@ -28,18 +28,15 @@ dependencies:
   google_sign_in: ^6.1.5
   url_launcher: ^6.1.14
   cached_network_image: ^3.3.0
-  uuid: ^3.0.6
+  uuid: ^4.1.0
   dio: ^5.3.3
   pretty_dio_logger: ^1.3.1
-  connectivity_plus: ^4.0.2
-  app_tracking_transparency: ^2.0.4
-  flutter_staggered_grid_view: ^0.7.0
```
Comment (lockieRichter) [inline pubspec.yaml:36]: Removed these 3 as we aren't using them anywhere

File: lib/src/ui/core/dante_components.dart (hunk 1 above)
Comment (shockbytes) [reply lib/src/ui/core/dante_components.dart]: I don't really care to be honest. What do you think?

**Review by shockbytes — APPROVED** [review]

File: lib/src/ui/core/dante_components.dart (hunk 1 above)
Comment (lockieRichter) [reply lib/src/ui/core/dante_components.dart]: I'll leave it for now, can always change later if we need.

Comment (lockieRichter) [reply lib/src/ui/core/dante_components.dart]: actaully I've changed it, then we don't need to pass around the build context so much

**Pull request merged**